packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarouselCard.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div :class="$style['c-restaurantCarouselCard']">
```

Comment:
Would it make sense to just call this `c-restaurantCard`? Seems like this could be used not just in a carousel potentially, soo keeping the naming generic would be good.

Reply:
@ashleynolan happy to change in follow-up PR? I did consider it but decided there might be a better `restaurantCard` than the carousel?

Reply:
The only other one I can think of would be the restaurantCard on SERP, but I imagine we would then extend this component so that we had a `c-restaurantCard--horizontal` class or something similar that just shifted the position of the elements.

I'd definitely update it to `c-restaurantCard` though as makes it more useful as a generic component that way.

Reply:
Perfect will get it updated